recipes/requests_mock/meta.yaml

```diff
@@ -0,0 +1,37 @@
+{% set version = "1.3.0" %}
+{% set name = "requests_mock" %}
+
+package:
+  name: "{{ name }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/r/requests_mock/requests-mock-{{ version }}.tar.gz
+  sha256: bd86970d6c52cc97071f5185aa594de6a997a5ca63b3bb36aceb9bb9db49294b
+
+build:
```

Comment:
This recipe might be a candidate for `noarch: python`:

https://conda-forge.readthedocs.io/en/latest/meta.html#building-noarch-packages